### ISSUE TITLE
elytra fly bot

### ISSUE THREAD

**Issue by DdaannnRU** [cabaletta/baritone#2931]
i`m found this client have an elytra fly bot, so it IS possible to release in baritone.
![2021-08-11_11 17 12](https://user-images.githubusercontent.com/88767403/128990182-f0c467df-ec9b-49bb-bc92-298d74a3ddfc.png)

https://github.com/bebeli555/CookieClient/releases

Things it may not ever have, from most likely to least likely =(

Boats
Horses (2x3 path instead of 1x2)
Elytra

**Comment by scorbett123** [comment]
duplicate of #2328
